packages/react-scripts/package.json

```diff
@@ -44,14 +44,14 @@
     "css-loader": "3.6.0",
     "dotenv": "8.2.0",
     "dotenv-expand": "5.1.0",
-    "eslint": "^6.6.0",
+    "eslint": "^7.3.0",
     "eslint-config-react-app": "^5.2.1",
-    "eslint-loader": "4.0.2",
-    "eslint-plugin-flowtype": "4.7.0",
-    "eslint-plugin-import": "2.20.2",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-react": "7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-flowtype": "^5.1.3",
+    "eslint-plugin-import": "^2.21.2",
```

Comment:
Thanks a lot for this PR. Any plan to release `react-scripts`? I really need to get a hotfix coming from `eslint-plugin-import`, now that `react-scripts` supports minor updates with `^` symbol.